ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -292,6 +292,7 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 				on('keydown', this.handleKeyDown);
 				on('keyup', this.handleKeyUp);
 			}
+			window.addEventListener('resize', this.handleResize);
```

Comment:
Let's check the existence of window.

```suggestion
			if (typeof window !== 'undefined') {
				window.addEventListener('resize', this.handleResize);
			}
```

Reply:
@mmyelyn @vJIYEv Please keep in mind that you should always check `window` for isomorphic + v8 snapshot build in Enact